docs/options.md

```diff
@@ -197,8 +193,34 @@ Optimize 42 SVG(s) with SVGO
 
 ---
 
+## Configuration Path
+
+| Name                 | Default Value               | Workflow file      | Config File |
+| -------------------- | --------------------------- | ------------------ | ----------- |
+| `configuration-path` | `".github/svgo-action.yml"` | :heavy_check_mark: | :x:         |
+
+The _configuration path_ option can be used to change the location of the
+[configuration file].
+
+### Examples
+
+To enable comments by the Action:
+
+```yaml
+# .github/workflows/svgo.yml
+- uses: ericcornelissen/svgo-action@latest
+  with:
+    configuration-path: path/to/configuration/file.yml
+```
+
+---
+
 ## Conventional Commits
 
+| Name                   | Default Value | Workflow file      | Config File |
+| ---------------------- | ------------- | ------------------ | ----------- |
+| `conventional-commits` | `true`        | :heavy_check_mark: | :x:         |
```

Comment:
Actually, the default value is `false` as per [the `action.yml` file](https://github.com/ericcornelissen/svgo-action/blob/be7d8d09dd3a3e0288e337ef6e5a4eb9f5b74dda/action.yml#L17-L20)

```suggestion
| `conventional-commits` | `false`       | :heavy_check_mark: | :x:         |
```